ksqldb-functional-tests/src/test/resources/query-validation-tests/elements.json

```diff
@@ -39,6 +39,19 @@
       "inputs": [{"topic": "input", "value": {"c1": 4}}],
       "outputs": [{"topic": "OUTPUT", "value": {"C1": 4}}]
     },
+    {
+      "name": "validate unwrapped value without elements OK - AVRO",
+      "statements": [
+        "CREATE STREAM INPUT WITH (kafka_topic='input', value_format='AvRo', wrap_single_value=false);",
+        "CREATE STREAM OUTPUT as SELECT * FROM input;"
+      ],
+      "topics": [
+        {"name": "input", "format": "AVRO", "schema": "int"},
+        {"name": "OUTPUT", "format": "AVRO", "schema": {"name": "blah", "type": "record", "fields": [{"name": "ROWVAL", "type": "int"}]}}
+      ],
+      "inputs": [{"topic": "input", "value": 4}],
+      "outputs": [{"topic": "OUTPUT", "value": {"ROWVAL": 4}}]
```

Comment:
does the output intentionally not keep the serde options from the input? I would expect it to be not wrapped here as well

Reply:
`wrap_single_value` is not currently inherited.  Not sure why it was implemented that way, but it was.  This PR doesn't change that.